dspace/config/dspace.cfg

```diff
@@ -32,8 +32,13 @@ dspace.baseUrl = http://localhost:8080
 # the context path to the UI you are using.
 #
 # Alternatively, you can use a url redirect or deploy the web application under the servlet container root.
+# This is the URL that the front-end will be served on
 dspace.url = ${dspace.baseUrl}
 
+# This is the URL that will be used for the REST endpoints to be served on.
+# This will typically be followed by /api to determine the root endpoints.
+dspace.restUrl = ${dspace.baseUrl}/rest
```

Comment:
The default DSpace instalation to work properly, this configuration should be redefined to:
dspace.restUrl = ${dspace.baseUrl}/spring-rest

or will the "spring-rest" be renamed  to "rest"?

Reply:
We've moved this to https://github.com/DSpace/DSpace/pull/2258
I think it would be best to discuss it there since this is not specific to entities